components/NavLink.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import Link from 'next/link'
+
+type NavLinkProps = {
+  text: string
+  path: string
+  activePath?: string
+  as?: string
+  internal?: true
```

Comment:
better to use `external` as a default.

Reply:
we want to default to internal link, then if someone wants to link to external they should explicitly say so.

Reply:
Yes, this makes more sense